content/how-to/work-calendars/about-calendars-and-overrides.md

```diff
@@ -0,0 +1,114 @@
+If it finds active definitions and equipment for that period, the service creates work calendar entries and persists the data to a time-series database.
+This topic explains how that calendar service works.
+
+{{< notice "note" >}}
+Rhize's implementation of work calendars was inspired by ISO/TR
+22400-10, a standard on KPIs in operations management. 
+{{< /notice >}}
+ 
+## What the service does
+
+{{< bigFigure
+src="/images/work-calendars/diagram-rhize-calendar-service-swimlane.png"
+alt="A simplified view of how the calendar service coordinates and exchanges data with the Rhize DB and a time-series DB"
+caption="A simplified view of how the calendar service coordinates and exchanges data with the Rhize DB and a time-series DB"
+width="80%"
+>}}
+
+
+The calendar service queries all active work calendar definitions at an interval designated in your [service configuration]({{< relref "/reference/service-config" >}}).
+The service then checks for any active `workCalendarDefinitionEntry` items that start or end within that interval.
+If any exist, Rhize creates a `workCalendarEntry` with the start and end time.
+
+The service then traverses all the calendar `hierarchyScope` entities (designated by the prefix `WorkCalendar`) and their `equipmentHierarchy` properties.
+Rhize checks each equipment item for any `workCalendarEntries`.
+If the scope includes active equipment, Rhize persists the entry to a time-series database.
+
+### The relationship between hierarchy scope, equipment, and calendars
+
+```mermaid
+flowchart TD
+hs(Hierarchy scope) -->|provides scope for| wc(Work calendar)
+e(Equipment) -->|physical structure maps to| hs
+hs --> |calendar structure maps to| e
+wc -->|create calendar entries for active|e
+```
+
+As the behavior in the previous section describes, the work calendar service coordinates data between 
+three entities in your knowledge graph: {{< abbr "hierarchy scope" >}}, {{< abbr "equipment" >}}, and {{< abbr "work calendar" >}}.
+These entities work together to configure work calendars and automate changes of equipment state.
+
+- **Equipment** provides the physical hierarchy of the plant's equipment, at levels that can be as small as a `workUnit` or as broad as the entire enterprise.
+- **Hierarchy scope** creates a calendar hierarchy that the equipment hierarchy maps to.
+- **Work calendars** and their associated definitions and entries have a hierarchy scope property, which Rhize uses to determine what the equipment state is.
+
+
+The Rhize service uses the hierarchy scope to establish calendar precedence.
+Then, it uses scope's associated calendar states to automatically set the state of the equipment for each hierarchy.
+So, when you [create a calendar]({{< relref "create-work-calendar" >}}), ensure that you configure these three objects.
+  
+  
+
+### Calendar states
+
+The Rhize database and service has three calendar types:
+
+- `PlannedBusyTime`
+- `PlannedDowntime`
+- `PlannedShutdown`
+
```

Comment:
The Rhize database and service has three calendar entry types:


- `PlannedDowntime`
- `PlannedShutdown`
-  'Other'

Other is used to place calendar events into the database which will not be considered when calculating OEE